source/code/plugin/in_kube_nodes.rb

```diff
@@ -188,7 +192,12 @@ def parse_and_emit_records(nodeInventory, batchTime = Time.utc.iso8601)
             properties["Computer"] = record["Computer"]
             properties["KubeletVersion"] = record["KubeletVersion"]
             properties["OperatingSystem"] = nodeInfo["operatingSystem"]
-            properties["DockerVersion"] = dockerVersion
+            properties["ContainerRuntimeVersion"] = containerRuntimeVersion
```

Comment:
just one thing is enough for telemetry. We will pay more by duplicating the versions in both 'dockerversion' and 'containerruntimeversion' dimensions for docker based clusters (which is our majority as of now). To not break our telemery queries, just put the runtime version in dockerVersion dimension.

Reply:
fixed. followed the same pattern as containernodeinventory i.e. DockerVersion fields just version if it is docker else <runtime>://<version>